tests/playwright/page-object-model/shared/search-form-component.ts

```diff
@@ -1,10 +1,13 @@
 import { Locator, Page, expect } from '@playwright/test'
 
-interface expectedResult {
-  name: string
-  address: string
-  ukprn: string
-  academiesInTrustCount: number
+class SelectedTrust {
+  name = ''
+  address = ''
+}
+
+export class CurrentSearch {
+  term = ''
+  selectedTrust = new SelectedTrust()
```

Comment:
If this is just for autocomplete, should we rename it to reflect that? So it doesn't get confused with trusts from fakedata

Reply:
It is also for if a user clicks on a trust in the search results list